node-graph/nodes/gstd/src/text.rs

```diff
@@ -25,7 +28,8 @@ fn text<'i: 'n>(
 	#[default(0.)]
 	tilt: f64,
 	align: TextAlign,
-	/// Splits each text glyph into its own row in the table of vector geometry.
+	/// When enabled, outputs a table with one Vector per character, allowing individual character manipulation.
```

Comment:
"one Vector" -> "one vector element"

Add "by subsequent operations" to the end of the last sentence.